docs/rst/use_cases/vulcanexus_cloud.rst

```diff
@@ -0,0 +1,150 @@
+a |ddsrouter| instance at each side of the communication.
+
+.. figure:: /rst/figures/ddsrouter_overview_wan.png
+
+Local setup
+===========
+The local instance of |ddsrouter| (local router) only requires to have a :ref:`Simple Participant <user_manual_participants_simple>`,
+and a :ref:`WAN Participant <user_manual_participants_wan>` that will play the client role in the discovery process of
+remote participants (see `Discovery Server <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/discovery_server.html>`_).
+
+After having acknowledged each other’s existence through `Simple DDS discovery mechanism <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/simple.html>`_
+(mutlicast communcation), the local participant will start receiving messages published by the ROS 2 talker node, and
+will then forward them to the WAN participant. Following, these messages will be sent to another participant hosted on a
+|k8s| cluster to which it connects via WAN communication over UDP/IP.
+
+Following is a representation of the above-described scenario:
+
+.. figure:: /rst/figures/vulcanexus_local.png
+
+
+Local router
+------------
+The configuration file used by the local router will be the following:
+
+.. literalinclude:: ../../resources/use_cases/vulcanexus_cloud/local-ddsrouter.yaml
+    :language: yaml
+
+Note that the simple participant will be receiving messages sent in DDS domain ``0``. Also note that, due to the choice
+of UDP as transport protocol, a listening address with the LAN public IP address needs to be specified for the local WAN
+participant, even when behaving as client in the participant discovery process. Make sure that the given port is
+reachable from outside this local network by properly configuring port forwarding in your Internet router device.
+The connection address points to the remote WAN participant deployed in the |k8s| cluster.
+
+To launch the local router from within a Vulcanexus Docker image, execute the following:
+
+.. code-block:: bash
+
+    docker run -it --net=host -v local-ddsrouter.yaml:/tmp/local-ddsrouter.yaml ubuntu-vulcanexus:galactic -r "ddsrouter -c /tmp/local-ddsrouter.yaml"
+
+
+Talker
+------
+*ROS 2* demo nodes is not installed in *Vulcanexus* distribution by default, but one can easily create a new Docker
+image including this feature by using the following Dockerfile:
+
+.. literalinclude:: ../../resources/use_cases/vulcanexus_cloud/Dockerfile_LAN
+    :language: Dockerfile
+
+Create the new image and start publishing messages by executing:
+
+.. code-block:: bash
+
+    docker build -t vulcanexus-demo-nodes:galactic -f Dockerfile .
+    docker run -it --net=host vulcanexus-demo-nodes:galactic -r "ros2 run demo_nodes_cpp talker"
+
+
+
+Kubernetes setup
+================
+Two different deployments will be used for this example, each in a different |k8s| pod. The |ddsrouter| cloud instance
```

Comment:
I would add an introduction sentence here. Something like:
In order to communicate our local scenario with a cloud k8s cluster...
In general, I would appreciate somewhere where it is explained the whole scenario of this example. A paragraph or a figure